src/sidebar/components/test/Thread-test.js

```diff
@@ -138,12 +140,6 @@ describe('Thread', () => {
       assert.calledOnce(fakeStore.setExpanded);
       assert.calledWith(fakeStore.setExpanded, replyThread.id, false);
     });
-
-    it('assigns an appropriate CSS class to the element', () => {
```

Comment:
Neither this nor the other test removed tested for anything particularly meaningful, IMHO.